README.md

```diff
@@ -1,110 +1,138 @@
+#### `resolvers.storage.endpoints`
+- `index` defines the location of the index file in sda-download. This is assumed to be non-encrypted in all cases.
+- `file` defines the location of the header and file requested from sda-download by htsget in order to calculate the ranges according to the request.
 
-In order to run the demo version of htsget, first copy `config-htsget/config-no-tls.json.example` to `config-htsget/config-no-tls.json`, then follow the instructions about removing tls at the bottom of this file and finally run
-```sh
-docker compose -f docker-compose-htsget.yml --profile demo up
-```
-Once all the services have completed, you should be able to find the `output` folder in the root of the repository, containing the `NA12878.bam` file.
 
-If you are interested in the commands used for extracting the data, you can check the `data_downloader` section of the `docker-compose-htsget.yml` file.
+The values in the config file `config-htsget-rs/download-config.toml` allow for requesting encrypted partial or full files.
 
-## Running the services - Starter kit version
-The htsget product of the starter kit depends on the storage-and-interfaces product. Specifically, the data served has to be ingested and stored in the archive included in the [storage-and-interfaces repository](https://github.com/GenomicDataInfrastructure/starter-kit-storage-and-interfaces). Therefore, in order to test the implementation, the two docker compose files (the storage-and-interfaces and the htsget) need to be started together, so that they share the same network and the different containers have access between them.
+## Running the services
+The htsget product of the starter kit depends on the storage-and-interfaces product. Specifically, the data served has to be ingested and stored in the archive included in the [storage-and-interfaces repository](https://github.com/GenomicDataInfrastructure/starter-kit-storage-and-interfaces). This is achieved by using the `docker-compose-demo.yml`, as below.
 
-Begin by copying `config-htsget/config.json.example` to `config-htsget/config.json`. To start the services, start the individual docker compose environments from their respective root directories:
+To start the services, start the individual docker compose environments from their respective root directories:
 ```sh
-starter-kit-storage-and-interfaces$ docker compose up -d
-starter-kit-htsget$ docker compose -f docker-compose-htsget.yml up -d
+starter-kit-storage-and-interfaces$ docker compose -f docker-compose-demo.yml up -d 
+starter-kit-htsget$ docker compose up -d
```

Comment:
```suggestion
docker compose up -d # in the folder starter-kit-htsget
```

Reply:
It will be easier for people to copy/paste the command. Probably even better to split into two blocks.